tests/functional/spec/features/routing/checkbox_count_routing.spec.js

```diff
@@ -0,0 +1,43 @@
+import ToppingCheckboxPage from "../../../generated_pages/new_routing_checkbox_count/topping-checkbox.page.js";
+import CorrectAnswerPage from "../../../generated_pages/new_routing_checkbox_count/correct-answer.page";
+import IncorrectAnswerPage from "../../../generated_pages/new_routing_checkbox_count/incorrect-answer.page";
+
+describe("Test routing using count of checkboxes checked", () => {
+  beforeEach(() => {
+    browser.openQuestionnaire("test_new_routing_checkbox_count.json");
+  });
+
+  it("Given a user selects 2 checkboxes, When they submit, Then they should be routed to the correct page", () => {
+    $(ToppingCheckboxPage.cheese()).click();
+    $(ToppingCheckboxPage.ham()).click();
+    $(ToppingCheckboxPage.submit()).click();
+
+    expect(browser.getUrl()).to.contain(CorrectAnswerPage.pageName);
+    expect($("body").getText()).to.have.string("You were asked to select 2 toppings and you selected 2.");
```

Comment:
I think fetching fetching from body should be avoiding, we should use a selector we have. If you don't have the selector for description, you can use the `questionTitle()` selector. You will need to update the title for `Correct` to like `You selected 2 toppings`. Then this would be like:
```js
expect($(CorrectAnswerPage.questionText()).getText()).to.have.string("You selected 2 toppings");
```